components/main-nav.vue

```diff
@@ -25,12 +29,12 @@ export default {
       default: null
     }
   },
-  computed: {
-    navItems() {
-      return this.mainNavItems.filter((navItem) => {
-        const [{ text }] = navItem.primary.title
-        return text !== 'Accueil'
-      })
+  methods: {
+    isPixProActive(index) {
+      if (index !== this.mainNavItems.length - 3) return false
```

Comment:
cette ligne qui return `false`, n'est-elle pas inclue dans le `return ['/employeurs', '/mediation-numerique'].includes(this.$nuxt.$route.path)` ?